titan/features/prep.py

```diff
@@ -54,13 +54,27 @@ def init_agent(self, pop: "population.Population", time: int):
             pop: the population this agent is a part of
             time: the current time step
         """
-        if (
-            not self.agent.hiv
-            and self.eligible()
-            and time >= self.agent.location.params.prep.start_time
-            and pop.pop_random.random() < self.agent.location.params.prep.target
-        ):
-            self.enroll(pop.pop_random, time)
+        params = self.agent.location.params
+        if not self.agent.hiv and self.eligible() and time >= params.prep.start_time:
```

Comment:
should probably move the hiv and prep start time checks into prep eligible instead of here.